contributors/guide/README.md

```diff
@@ -218,7 +218,8 @@ To make it easier for your PR to receive reviews, consider the reviewers will ne
 * break large changes into a logical series of smaller patches which individually make easily understandable changes, and in aggregate solve a broader issue
 * label PRs with appropriate SIGs and reviewers: to do this read the messages the bot sends you to guide you through the PR process
 
-Reviewers, the people giving the review, are highly encouraged to revisit the [Code of Conduct](/code-of-conduct.md) and must go above and beyond to promote a collaborative, respectful community.  
+Reviewers, the people giving the review, are highly encouraged to revisit the [Code of Conduct](/code-of-conduct.md) and must go above and beyond to promote a collaborative, respectful community.
+As Github has introduced the status in the user account, now the bot uses this status for auto-assigning reviewers, for example if a reviewer has a status set to "Busy" so the bot will not assign them new PRs for review.  
```

Comment:
Instead of repeating this line here, I think it would be good to link the code review section in the `community-expectations.md` doc here.

Reply:
done 👍, you can check now